src/standard-live-connect.ts

```diff
@@ -119,28 +119,44 @@ function _standardInitialization (liveConnectConfig: LiveConnectConfig, external
   }
 }
 
+function _initializeWithoutGlobalName(liveConnectConfig: LiveConnectConfig, externalStorageHandler: ExternalStorageHandler, externalCallHandler: ExternalCallHandler, eventBus: EventBus): ILiveConnect {
+  const lc = _standardInitialization(liveConnectConfig, externalStorageHandler, externalCallHandler, eventBus)
+  window.liQ_instances = window.liQ_instances || []
+  window.liQ_instances.push(lc)
+  return lc
+}
+
+function _initializeWithGlobalName(liveConnectConfig: LiveConnectConfig, externalStorageHandler: ExternalStorageHandler, externalCallHandler: ExternalCallHandler, eventBus: EventBus): ILiveConnect {
+  const queue = window[liveConnectConfig.globalVarName] || []
+  const lc = _getInitializedLiveConnect(liveConnectConfig) || _standardInitialization(liveConnectConfig, externalStorageHandler, externalCallHandler, eventBus) || queue
+  
+  if (isArray(queue)) {
+    for (let i = 0; i < queue.length; i++) {
+      lc.push(queue[i])
+    }
+  }
+  
+  window[lc.config.globalVarName] = lc
+
+  window.liQ_instances = window.liQ_instances || []
+  if (window.liQ_instances.filter(i => i.config.globalVarName === lc.config.globalVarName).length === 0) {
+    window.liQ_instances.push(lc)
+  }
```

Comment:
Let's factor this out as it's repeated 4 timers

Reply:
In one of the intermediate versions, I had it factored out. Decided to do it that way on purpose: in the next step all logic that initializes LC with globalVarName will move into the builder project. Thus I can simple remove the _initializeWithGlobalName functions.